homeassistant/components/lcn/helpers.py

```diff
@@ -247,6 +247,76 @@ def import_lcn_config(lcn_config: ConfigType) -> list[ConfigType]:
     return list(data.values())
 
 
+def purge_entity_registry(
+    hass: HomeAssistant, entry_id: str, entry_data: ConfigType
```

Comment:
The last parameter is the imported config data, not really the config entry data. The new config entry isn't created yet. I'd rename the parameter.